setup.py

```diff
@@ -1,5 +1,9 @@
 from setuptools import setup, find_packages
-import pathlib
+
+try:
+    import pathlib
+except ImportError:
+    import pathlib2 as pathlib
```

Comment:
I suggest to use `os` instead of requiring a dependency
```suggestion
    import os
```